### PR TITLE
remove ContainerOp from test cases - part 3

### PR DESCRIPTION
**Pull request by yhwang**
eliminate the usage of ContainerOp in testcases and
update the testcase accordingly, including filenames starting
with `o` to `u`

related to #670

Signed-off-by: Yihong Wang <yh.wang@ibm.com>

### CODE REVIEW

File: sdk/python/tests/compiler/testdata/pipelineparams.yaml [hunk 1]
```diff
@@ -45,48 +45,43 @@ spec:
...
-        value: $(params.sleep_ms)
-      - name: tag
-        value: $(params.tag)
       taskSpec:
         steps:
         - name: main
           args:
-          - sleep $(inputs.params.sleep_ms); wget localhost:5678 -O $(results.downloaded-resultoutput.path)
+          - |
+            sleep None; wget localhost:5678 -O $0
```
Comment (Tomcli) [inline sdk/python/tests/compiler/testdata/pipelineparams.yaml:53]: it should be `sleep $(inputs.params.sleep_ms);`. Do you think valuable replacement for image works in component.yaml?

File: sdk/python/tests/compiler/testdata/retry.py [hunk 2]
```diff
@@ -12,23 +12,33 @@
...
+      - {name: exitcode, type: String}
+    implementation:
+      container:
+        image: python:alpine3.6
+        command:
+        - python
+        - -c
+        args:
+        - |
+          import random; import sys; \
```
Comment (Tomcli) [inline sdk/python/tests/compiler/testdata/retry.py:33]: do you think we need a new line here? when compiling it, the newline makes the spacing look very bad.

File: sdk/python/tests/compiler/testdata/sequential_noninlined.yaml [hunk 3]
```diff
@@ -74,7 +85,7 @@ spec:
         steps:
         - name: main
           args:
-          - echo "$0"
+          - echo
```
Comment (Tomcli) [inline sdk/python/tests/compiler/testdata/sequential_noninlined.yaml:88]: quick question, does this command still work without "$0"?

Comment (yhwang) [reply sdk/python/tests/compiler/testdata/sequential_noninlined.yaml:88]: yes, for the old one, it uses parameter substitution. the new one just two arguments. both produce the same output

File: sdk/python/tests/compiler/testdata/pipelineparams.yaml (hunk 1 above)
Comment (yhwang) [reply sdk/python/tests/compiler/testdata/pipelineparams.yaml:53]: I guess it doesn't work. then we need to fix the compiler error. let me revert this and put a comment on issue 670

File: sdk/python/tests/compiler/testdata/retry.py (hunk 2 above)
Comment (yhwang) [reply sdk/python/tests/compiler/testdata/retry.py:33]: sure, merge it with next line

**Pull request merged**